main.js

```diff
@@ -56,54 +72,80 @@
 
   function clearFields() {
     if (challengerNames[0].hasAttribute('disabled')) {
-        challengerInputField[1].value = ' '
-        challengerInputField[3].value = ' '
-      }else {
-        for (var i = 0; i < challengerInputField.length; i++) {
-          challengerInputField[i].value = ' '
+      challengerInputField[1].value = ' '
+      challengerInputField[3].value = ' '
+    } else {
+      for (var i = 0; i < challengerInputField.length; i++) {
+        challengerInputField[i].value = ' '
       }
     }
-    clearButton.setAttribute('disabled', true);
+
     submitButton.setAttribute('disabled', true);
   }
 
   function displayChallengerInputs() {
     playerOneDisplay.innerText = challengerNames[0].value;
     playerTwoDisplay.innerText = challengerNames[1].value;
-    for (var i = 0; i < challengerNames.length; i++) {
-      challengerNames[i].setAttribute('disabled', true);
-    };
-    checkGuess1();
-    checkGuess2();
+    disableInput()
+    checkGuesses()
     guessOneDisplay.innerText = challengerGuesses[0].value;
     guessTwoDisplay.innerText = challengerGuesses[1].value;
-      challengerGuesses[0].value = ' '
```

Comment:
created new functions to break down existing functions into smaller bites/fewer lines of code.